src/elements/InputText.vue

```diff
@@ -98,6 +145,14 @@ export default {
       default: "",
       required: true,
     },
+    /**
+     * Value passed when submitting a form.
```

Comment:
This prop is self-evident, but I wonder if the description is confusing since there's also a `value` prop that gets passed and is actually the value of the input. Perhaps just "The name of the <input> element." ?

Reply:
What about "The name attribute for the form input field" or "Input data passed when submitting a form"?